WolvenKit.RED4.Types/ClassesExt/Properties/cpGameplayLightComponent.cs

```diff
@@ -4,10 +4,10 @@ public partial class cpGameplayLightComponent
     {
         [OrdinalOverride(Before = 9)]
         [RED("placedEditorData")]
-        public CHandle<entIPlacedComponentEditorData> PlacedEditorData
+        public new CHandle<entIPlacedComponentEditorData> PlacedEditorData
         {
             get => GetPropertyValue<CHandle<entIPlacedComponentEditorData>>();
-            set => SetPropertyValue<CHandle<entIPlacedComponentEditorData>>(value);
+            set => SetPropertyValue(value);
```

Comment:
Even if it isn't needed, please revert so it matches the rest again :)

Reply:
Hm I don't remember doing that. Might've been some automatic fix by Rider or something I clicked. I'll make sure to revert it.